### DISCUSSION TITLE
Live camera support

### DISCUSSION THREAD

**Discussion by deeaamon** [jellyfin/jellyfin#7629] [Q&A]
Hi guys,  im not sure if this is possible now or not but I have been looking.  I am trying to figure out hot to use a live webcam with the Live TV.  I  thought it would be kind neat to do my own live shows for people on my Jellyfin  i tried setting up OBS or VLC to create a m3u playlist that is streamed to but it doesn't work.  I'm still very new to this so please forgive me if I'm not being clear.  The bottom line is I would like to stream a camera directy to jellyfin and people who access my jellyfin server can see it just like the movies and videos.  is this possible?

**Comment by PrplHaz4** [comment]
m3u with rtsp sources should work without issues (other than ~3s delay)

**Reply by deeaamon** [reply]
Yeah it just simply doesn't work.  It works if I access it from VLC Player using he URL "http://131.153.160.82:8097/live/test/index.m3u8"  but not from jellyfin when I add the URL to the Live TV runner devices as a mcu.

Let me try to explain what I'm trying to do and you guys cna let me know if it can be done.

1.  I set up a Node Media Server on my web server at http://131.153.160.82:8097 and rtmp://131.153.160.82:1935
2. I then used OBS Studio on my PC to stream my webcam to rtmp://131.153.160.82:1935/live with a stream key of 'test'
NOTE: This works.  If I go to http://131.153.160.82:8097/admin and view my streams, I can click on the video and see it streaming my webcam.  I know its not secure yet.  I'll work on that later. Also at this point I can access the http://131.153.160.82:8097/live/test/index.m3u8 from VLC player.
3.  I open up my dashboard for Jelly fin and go to live tv and add in the M3U tuner device http://localhost:8097/live/test/index.m3u8 and it allows me to add it.
4. I go to the home page of jellyfin and I now have an icon for Live TV. i click on it  and go to channels.  It hten shows me three boxes I can click.  Why 3 i don't know I thought it should only be 1.  When I click on any of them and click play it opens just a black screen with a loading icon.  But it never loads any video.  and in my console I get  "HLS Error: Type: networkError Details: levelLoadTimeOut Fatal: false"  

So it appears I don't know exactly what I'm doing.  it seems like it should work but I may be trying something here that jellyfin doesn't allow.  any ideas?

Thanks
Darrell

**Comment by thomasdbnet** [comment]
make your own .M3U8 file put it on the harddisk of the jellyfin server and add that one in the m3u tuner list in webinterface . browse to the file


example of .m3u8 file content: 


#EXTM3U 
#EXTVLCOPT--http-reconnect=true
#EXTINF:-1 tvg-chno="001" tvg-name="xxx",xxx
#EXTVLCOPT:program=001
http://xxx:xxx@10.0.0.321/video/mjpg.cgi
#EXTINF:-1 tvg-chno="001" tvg-name="xxxl",xxx
#EXTVLCOPT:program=003
http://xxx:xxx@10.0.0.123/img/video.mjpeg

it seems to work fine for me ;)

**Reply by thomasdbnet** [reply]
sorry i misreaden it. this is a way to add  mjpeg security camera's  in jellyfin as channels not a way to output  obs stream.  maybe it works alike ?

**Reply by AngeloBestetti** [reply]
It is possible from RTSP? I´m seeing just http.
